render/json.go

```diff
@@ -128,24 +130,9 @@ func (r JsonpJSON) Render(w http.ResponseWriter) (err error) {
 		return err
 	}
 
-	callback := template.JSEscapeString(r.Callback)
-	if _, err = w.Write(bytesconv.StringToBytes(callback)); err != nil {
-		return err
-	}
-
-	if _, err = w.Write(bytesconv.StringToBytes("(")); err != nil {
-		return err
-	}
-
-	if _, err = w.Write(ret); err != nil {
-		return err
-	}
-
-	if _, err = w.Write(bytesconv.StringToBytes(");")); err != nil {
-		return err
-	}
-
-	return nil
+	jsonpData := fmt.Sprintf("%s(%s);", template.JSEscapeString(r.Callback), bytesconv.BytesToString(ret))
+	_, err = w.Write(bytesconv.StringToBytes(jsonpData))
```

Comment:
Don't use fmt.Sprintf

```go
	_, err = w.Write(bytesconv.StringToBytes(template.JSEscapeString(r.Callback)+"("+bytesconv.BytesToString(ret)+");"))
```

Reply:
It's updated and here is the latest benchmark result
```
BenchmarkJsonpJSONRender(New Pull Request)
BenchmarkJsonpJSONRender         6468048               964.3 ns/op           824 B/op         10 allocs/op
BenchmarkJsonpJSONRender-2       6673245               900.2 ns/op           816 B/op         10 allocs/op
BenchmarkJsonpJSONRender-4       6562498               903.5 ns/op           821 B/op         10 allocs/op
BenchmarkJsonpJSONRender-8       6615834               910.8 ns/op           819 B/op         10 allocs/op
BenchmarkJsonpJSONRender-16      6635358               908.3 ns/op           818 B/op         10 allocs/op
```